classes/class-sailthru-content.php

```diff
@@ -52,16 +52,25 @@ public function init_settings() {
 		// Only show these fields if the status has been enabled
 		if ( isset ( $options['sailthru_content_api_status'] ) && 'true' === $options['sailthru_content_api_status'] ) {
 			
+			add_settings_field(
+				'sailthru_spider_status',
+				__( 'Spider', 'text_domain' ),
+				array( $this, 'render_sailthru_spider_status_field' ),
```

Comment:
suggestion: array( $this, 'render_sailthru_spider_status_field' ) => [$this, 'render_sailthru_spider_status_field' ]

If php's short array syntax works with Wordpress, can we use it here?
Can we update the other references as well?
https://www.php.net/manual/en/language.types.array.php

Reply:
The reason I used parentheses is because the rest of the add_settings_field functions in this file follow the same formatting (check out the one for sailthru_content_post_types below or the one for sailthru_content_api_status above)

Reply:
yeah, I hear you. Part of improving the code is also following the "Boy Scout Rule". Basically you want to leave the code better than you found it. You are not trying over haul the code, but any small improvement can go along way and is always appreciated. If you read the book Clean Code, the author writes about this concept and why it's important. If we could update this array syntax and the other array syntax as well that would be great!